Types don't support activeTour or Evented properties.

I have a very simple test with the current version 4.6.0:

```
import Shepherd from "shepherd.js";

let activeTour = Shepherd.activeTour; 
let evented = new Shepherd.Evented();
```
This will result in two errors on Typescript@3.5.3 
```
error TS2339: Property 'activeTour' does not exist on type 'typeof Shepherd'
error TS2339: Property 'Evented' does not exist on type 'typeof Shepherd'
```

Additionally the current types will let few things will compile which should not:
```
let activeTour = new Shepherd().activeTour;
new Shepherd().on("event", function() { ... });
```

It looks to me that these errors are mostly related to this recent commit [9466a97](https://github.com/shipshapecode/shepherd/commit/9466a9753f32297c0e32d6e081692b82261f6cba)

My suggestion for a fix that restores the documented behavior:

```
// shepherd.d.ts
import _Evented from './evented';
import _Step from './step';
import _Tour from './tour';

declare namespace Shepherd {
    export import Tour = _Tour;
    export import Step = _Step;
    export import Evented = _Evented;
    export const activeTour: _Tour | undefined;

    export function on(event: string, handler: Function): void;
    export function once(event: string, handler: Function): void;
    export function off(event: string, handler: Function): boolean | void;
    export function trigger(event: string): void;
}

export default Shepherd;
```
This also requires adding an empty namespace to evented.d.ts.
```
declare namespaces Evented { }
```
Unfortunately I could not find any way to get the desired result without duplicating the Evented interface which seems to be related to this limitation in Typescript: [#18163](https://github.com/microsoft/TypeScript/issues/18163).

--- a/src/types/shepherd.d.ts
+++ b/src/types/shepherd.d.ts
@@ -1,12 +1,11 @@
-import _Evented from './evented';
 import _Step from './step';
 import _Tour from './tour';
 
-declare class Shepherd extends _Evented {
-  activeTour?: _Tour;
-}
-
 declare namespace Shepherd {
+  /**
+   * The currently active tour instance
+   */
+  export const activeTour: _Tour | undefined;
   export import Step = _Step;
   export import Tour = _Tour;
 }

